source/_includes/heads_up.liquid

```diff
@@ -1,4 +1,4 @@
 <div class="sl-c-callout sl-c-callout--warning">
   <h3>⚠️ Heads up!</h3>
-  {{ contents | markdown }}
```

Comment:
Removing because in the Style Rules documentation we want to pass in a code example into a heads up section